### DISCUSSION TITLE
Welcome to Perplexity API Discussions!

### DISCUSSION THREAD

**Discussion by shubhang98** [perplexityai/api-discussion#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by rlburr** [comment]
I can't find a selection in Library for making a page, nor anywhere else.

**Comment by ArchanaCIAI** [comment]
Does Perplexity AI API support extracting insights from charts or graphs in image/pdf format?

**Comment by irakeshsharma** [comment]
Does Perplexity AI API support extracting metadata from image?
